src/main/kotlin/no/nav/omsorgspenger/personopplysninger/HentFamilieRelasjon.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.omsorgspenger.personopplysninger
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.TextNode
+import kotlinx.coroutines.runBlocking
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helse.rapids_rivers.River
+import no.nav.k9.rapid.behov.Behovsformat
+import no.nav.k9.rapid.river.BehovssekvensPacketListener
+import no.nav.k9.rapid.river.leggTilLøsning
+import no.nav.k9.rapid.river.requireArray
+import no.nav.k9.rapid.river.skalLøseBehov
+import org.slf4j.LoggerFactory
+
+internal class HentFamilieRelasjon(
+        rapidsConnection: RapidsConnection,
+        internal val familieRelasjonMediator: FamilieRelasjonMediator) : BehovssekvensPacketListener(
+        logger = LoggerFactory.getLogger(HentFamilieRelasjon::class.java)
+) {
+
+    init {
+        River(rapidsConnection).apply {
+            validate { packet ->
+                packet.skalLøseBehov(BEHOV)
+                packet.require(IDENTITETSNUMMER) { it.require() }
+                packet.require(TIL){ it.requireArray { entry -> entry is TextNode } }
+            }
+        }.register(this)
+    }
+
+    override fun handlePacket(id: String, packet: JsonMessage): Boolean {
+        logger.info("Mottatt $BEHOV").also { incMottattBehov(BEHOV) }
+
+        val identitetsnummer = packet[IDENTITETSNUMMER].asText()
+
+        val til = (packet[TIL] as ArrayNode)
+                .map { it.asText() }
+                .toSet()
+
+        val løsning = hentFamilieRelasjonerFor(
+                identitetsnummer = identitetsnummer,
+                correlationId = packet[Behovsformat.CorrelationId].asText())
+
+        packet.leggTilLøsning(
+                behov = BEHOV,
+                løsning = løsning
+        )
+
+        return true
+    }
+
+    override fun onSent(id: String, packet: JsonMessage) {
+        logger.info("Løst behov $BEHOV").also { incLostBehov(BEHOV) }
+    }
+
+    private fun hentFamilieRelasjonerFor(identitetsnummer: String, correlationId: String) = try {
+        runBlocking {
+            familieRelasjonMediator.hentFamilieRelasjon(
+                identitetsnummer = identitetsnummer,
+                correlationId = correlationId)
+        }
+    } catch (cause: Throwable) {
+        incBehandlingFeil(BEHOV)
+        throw cause
+    }
+
+    internal companion object {
+        const val BEHOV = "HentFamilierelasjoner"
```

Comment:
Tror `VurderRelasjoner` er mer riktig. For kan jo være at noen i `til`-listen ikke er familie, at relasjon=`INGEN` men `borSammen=true` for eksempel

Reply:
Fixet